static/app/components/feedback/widget/feedbackModal.tsx

```diff
@@ -133,6 +149,20 @@ export function FeedbackModal({title, type, children}: FeedbackModalProps) {
 }
 
 const Dialog = styled('dialog')`
+  --sentry-feedback-bg-color: #fff;
+  --sentry-feedback-bg-accent-color: #eee;
+  --sentry-feedback-fg-color: #000;
+  --sentry-feedback-border: 1.5px solid rgba(41, 35, 47, 0.13);
+  --sentry-feedback-box-shadow: 0px 4px 24px 0px rgba(43, 34, 51, 0.12);
+
+  &.__sntry_fb_dark {
```

Comment:
`fb` always reads as facebook to me.

`fdbk`?

Reply:
I'm taking it back

Reply:
I'll compromise on `fbk`

Reply:
kk, i don't really mind in any case. i'll let jesse take a look, unless you want it sooner than i can click the button

Reply:
Is it too bold to just prefix variables with `--sentry`? 

From the users' perspective, this widget is the only "injected code" by Sentry. Makes the naming less brittle because we're still not 100% sure what we're going to call this product.

Reply:
both naming schemes are subject to change when they make it into the SDK. e.g. if the form is a web component, we can name it whatever we want